pmm-app/tests/DbaaS/api/dbaasAPI.js

```diff
@@ -5,7 +5,10 @@ const { I } = inject();
 module.exports = {
 
   async apiRegisterCluster(clusterConfiguration, clusterName) {
-    const body = JSON.stringify({ kubernetes_cluster_name: `${clusterName}`, kube_auth: { kubeconfig: `${clusterConfiguration}` } });
+    const body = JSON.stringify({
```

Comment:
is there a reason to use `JSON.stringify()` here? I think it should work without it. (it works in other places)

Reply:
so I don't need to worry about the format endpoint needs and spend more time on checking what test-code is meant to check.